app/models/concerns/hostext/search.rb

```diff
@@ -170,6 +171,24 @@ def search_by_hostgroup_and_descendants(key, operator, value)
         {:conditions => opts}
       end
 
+      def search_by_params(key, operator, value)
+        key_name = key.sub(/^.*\./, '')
```

Comment:
I would rather see following as it behaves more predictable in situations for 'asdf.asdf.asdf'
```suggestion
        key_name = key.split('.').last
```